src/main/scala/nl/jqno/secretaresse/scalafx/SecretaressePresenter.scala

```diff
@@ -0,0 +1,76 @@
+package nl.jqno.secretaresse.scalafx
+
+import java.awt.Toolkit._
+import java.net.URL
+import javax.swing.SwingUtilities
+
+import com.typesafe.scalalogging.StrictLogging
+import nl.jqno.secretaresse.{Scheduler, Secretaresse, Tray}
+
+import scala.collection.immutable.ListMap
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+import scala.util.{Failure, Success}
+import scalafx.application.{JFXApp, Platform}
+import scalafx.geometry.Insets
+import scalafx.scene.Scene
+import scalafx.scene.layout.HBox
+import scalafx.scene.paint.Color._
+import scalafx.scene.text.Text
+
+object SecretaressePresenter extends JFXApp with StrictLogging {
+
+  val secretaresse = new Secretaresse("application.conf")
+
+  val scheduler = Scheduler(run())
+
+  def run(): Unit = {
+    secretaresse.sync() onComplete {
+      case Success(()) => // do nothing
+      case Failure(e) => Tray().notifyUser("Synchronising failed :(")
+    }
+  }
+
+  Platform.implicitExit_=(false)
```

Comment:
Volgens mij kun je dit ook schrijven als: `Platform.implicitExit = false`
Want Scala is magic!